build.gradle.kts

```diff
@@ -113,6 +113,13 @@ publishing {
     }
 }
 
+tasks.withType<Jar> {
```

Comment:
Same question re: `configureEach` as on the the CCUD PR.

Reply:
Yes. It should be `configureEach`. Fixed in most recent changes. b6374fd77df8f9e7d10b6f2dade6727ed2f3055b